xwiki-commons-core/xwiki-commons-xml/src/main/java/org/xwiki/xml/internal/html/DefaultHTMLCleaner.java

```diff
@@ -237,34 +236,35 @@ private CleanerProperties getDefaultCleanerProperties(HTMLCleanerConfiguration c
      */
     private CleanerTransformations getDefaultCleanerTransformations(HTMLCleanerConfiguration configuration)
     {
-        CleanerTransformations defaultTransformations = new CleanerTransformations();
+        CleanerTransformations defaultTransformations = new TrimAttributeCleanerTransformations();
 
-        TagTransformation tt = new TagTransformation(HTMLConstants.TAG_B, HTMLConstants.TAG_STRONG, false);
+        TrimAttributeTagTransformation tt = new TrimAttributeTagTransformation(HTMLConstants.TAG_B,
+            HTMLConstants.TAG_STRONG, false);
         defaultTransformations.addTransformation(tt);
 
-        tt = new TagTransformation(HTMLConstants.TAG_I, HTMLConstants.TAG_EM, false);
+        tt = new TrimAttributeTagTransformation(HTMLConstants.TAG_I, HTMLConstants.TAG_EM, false);
```

Comment:
Why do we need to use our own transformation class instead of the HTMLCleaner one? Don't we need our custom one only for INPUT elements?

Reply:
No we don't. As I mentioned: we skip trimming the attributes, so we need to transform ALL the attributes, except the input one.

Reply:
Changed that: actually we don't need it since the attribute are not preserved in those.